spec/features/user/user_show_spec.rb

```diff
@@ -0,0 +1,31 @@
+user_show_spec.rb
```

Comment:
- [ ] Consider deleting this one too, the same issue as above. Screenshot attached below 👇🏼:
![error_2](https://user-images.githubusercontent.com/45002654/141236037-512fb31e-21ee-43d2-87b5-cf0a8c3f7e5d.png)